classes/class-uninstall.php

```diff
@@ -58,21 +58,8 @@ public function uninstall() {
 			$this->plugin->settings->network_options_key,
 		);
 
-		// Verify current user's permissions before proceeding.
-		if ( ! current_user_can( $this->plugin->admin->settings_cap ) ) {
-			wp_die(
-				esc_html__( "You don't have sufficient privileges to do this action.", 'stream' )
-			);
-		}
-
-		if ( defined( 'DISALLOW_FILE_MODS' ) && true === DISALLOW_FILE_MODS ) {
-			wp_die(
-				esc_html__( "You don't have sufficient file permissions to do this action.", 'stream' )
-			);
-		}
-
 		// Prevent this action from firing.
-		remove_action( 'deactivate_plugin', array( 'Connector_Installer', 'callback' ), null );
+		remove_action( 'deactivate_plugin', array( 'WP_Stream\Connector_Installer', 'callback' ), null );
```

Comment:
This was leaving a deactivation log in the DB for the blog where the plugin was being uninstalled because of the missing fully-qualified namespace.

Reply:
That's a great idea! Will change that up.